bagitobjecttransfer/recordtransfer/urls.py

```diff
@@ -58,6 +62,21 @@
         login_required(views.SubmissionCsv.as_view()),
         name="submissioncsv",
     ),
+    path(
+        "submission_group/new",
+        login_required(views.SubmissionGroupCreateView.as_view()),
+        name="submissiongroupnew",
+    ),
+    path(
+        "submission_group/<uuid:uuid>/",
+        login_required(views.SubmissionGroupDetailView.as_view()),
+        name="submissiongroupdetail",
+    ),
+    path(
+        "user/<int:user_id>/group-descriptions/",
+        login_required(views.get_users_group_descriptions),
+        name="get_users_group_descriptions",
+    ),
```

Comment:
Suggestion to make the URL a bit more uniform with the other URLs:

```suggestion
    path(
        "user/<int:user_id>/submission_groups/",
        login_required(views.get_user_submission_groups),
        name="user_submission_groups",
    ),
```

Reply:
I am using this path to send back specifically submission group **descriptions**. Would it be better to send back entire groups instead?

Reply:
Resolved by: https://github.com/NationalCentreTruthReconciliation/Secure-Record-Transfer/pull/226/commits/b60e49d6b068284632a665faf533ec11b1fd0a0b